ci/docker/image-builder/src/main/kotlin/ru/avito/image_builder/internal/ImageBuildProperties.kt

```diff
@@ -0,0 +1,42 @@
+package ru.avito.image_builder.internal
+
+import java.io.File
+import java.util.Properties
+
+/**
+ * Properties file to store associated attributes for image
+ */
```

Comment:
The same idea as before.
It's used to store metadata in the code rather than passing all by arguments.

```
./publish.sh <directory with Dockerfile>
```

Changed format from YAML to properties. 
Yaml is excessive and requires external dependencies to parse.

Reply:
I'm thinking about removing this feature.

Motivation:
- These configs already are lying. E.g. `android-builder` has different names in the in-house registry and Docker hub. It means, that these configs are not the source of truth and make more confusion. Maybe, it can be fixed and restricted, but I don't see why.
- Having simple and straightforward logic is more important for me than explicitly storing one additional argument in CI scripts.

Reply:
#1341